### ISSUE TITLE
Provide this font in TTF format

### ISSUE THREAD

**Issue by noscript** [sunaku/tamzen-font#7]
Where is `ttf/` subfolder?

**Comment by sunaku** [comment]
Nice catch :+1:  and it looks like the `pcf/` folder wasn't checked in either.  :sweat: 

I'll need time to get my new machine set up to regenerate the fonts. :turtle:

In the meantime, you could try regenerate them yourself (see instructions at bottom of README). :construction_worker:

**Comment by sunaku** [comment]
I've checked in the `pcf/` folder now in commit 445716dbc8f7586d5df4e1218c9580c0da4c8191. :gift:

However, it seems that FontForge simply doesn't generate any output files for the `ttf/` folder. :sweat:

**Comment by sunaku** [comment]
I've abandoned all hope of generating TTFs with FontForge, but [this looks promising](https://github.com/koron/bdf2ttf). :pray:

**Comment by noscript** [comment]
I will give it a try.

**Comment by sunaku** [comment]
Well, I gave it a shot but encountered an error: https://github.com/koron/bdf2ttf/issues/1 :grimacing:

**Comment by sunaku** [comment]
Thanks to @koron, we're finally making progress:

![screenshot](http://i.imgur.com/bjwGixC.png)

See also https://github.com/koron/bdf2ttf/issues/2

**Comment by noscript** [comment]
Can't wait :smile:

**Comment by sunaku** [comment]
Hit a wall with `bdf2ttf` :facepunch:, trying FontForge again: https://github.com/fontforge/fontforge/issues/2738

**Comment by vasili111** [comment]
Waiting ttf too :)

**Comment by sunaku** [comment]
Just to clarify: I'm waiting for feedback from FontForge developers in issue https://github.com/fontforge/fontforge/issues/2738.
If you're knowledgeable about TTF or passionate about this issue, feel free to help move this forward. :bowing_man:

**Comment by tobia** [comment]
I managed to convert it very easily using https://github.com/ali1234/bitmap2ttf but I'm having an issue with the space character width.

The reason I'm trying to create a TTF file is that AFAIK Sublime Text does not supports X11 fonts. The TTF file I created seems to have the correct width (hard tabs line up correctly) but the space character does not. A single space seems to be rendered 5 pixels wide instead of 8 (using 8x15):

![space_width](https://user-images.githubusercontent.com/48756/30583571-b31c8e44-9d27-11e7-9d7a-4d946a9cbb19.png)

I get the same results with the Tamsyn PCF files. Does anybody know how to fix it?

**Comment by sunaku** [comment]
Thanks for the suggestion @tobia! :+1:  That tool is generating "blank" SVGs for space characters:

```svg
/tmp/tmprKeQ4w/00032.svg:<svg width="500.0" height="1000"><path d="" fill="currentColor"/></svg>
/tmp/tmprKeQ4w/00160.svg:<svg width="500.0" height="1000"><path d="" fill="currentColor"/></svg>
```

As a result, my fontforge (version 11-Aug-2017) is refusing to process those two "blank" SVGs:

```sh
$ fontforge -script /tmp/tmprKeQ4w/Tamzen8x16r.ttf.pe
Copyright (c) 2000-2014 by George Williams. See AUTHORS for Contributors.
 License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
 with many parts BSD <http://fontforge.org/license.html>. Please read LICENSE.
 Based on sources from 13:46 UTC 11-Aug-2017-ML-D.
 Based on source from git with hash:
I'm sorry this file is too complex for me to understand (or is erroneous)
```

It seems your Fontforge did not reject the conversion outright and instead gave you a zero-width blank glpyh for the space character (codepoint 32).  That's why you're observing that issue in your screenshot. :sweat_smile:

**Comment by vasili111** [comment]
[Here](https://int10h.org/blog/2018/05/flexi-ibm-vga-scalable-truetype-font/) guy transformed bitmap to ttf. There are some clues but no tutorial.

**Comment by vasili111** [comment]
There is a response in comments about how to convert fonts. https://int10h.org/blog/2018/05/flexi-ibm-vga-scalable-truetype-font/

**Comment by sunaku** [comment]
Thanks @vasili111, I've applied your suggestion in [the `issue-7` branch]( https://github.com/sunaku/tamzen-font/tree/issue-7 ) and also added the generated `*.ttf` files there (under `ttf/` folder).  Please try it out and let me know if you find any problems.

**Comment by sunaku** [comment]
Good news, I have verified that the TTF fonts work correctly under Windows 10 build 1909:

![2020-05-07 19_52_06-](https://user-images.githubusercontent.com/9863/81365731-cb1a8700-909d-11ea-8640-e4de16db357a.png)

The fix is now merged into master branch at commit c1664cb26f7b68e736ab1f227abac86ae126e7e1.  :tada: Many thanks @vasili111!

**Comment by vasili111** [comment]
@sunaku 

Thank you very much for creating TTF versions :)
